airbyte-integrations/connectors/source-yandex-metrica/source_yandex_metrica/schemas/sessions.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
should this be empty?

Reply:
Yes. Schema is generated dynamically with `get_json_schema` method.